### DISCUSSION TITLE
“Age of data” switch for the “SELL” and “BUY” possible ?

### DISCUSSION THREAD

**Discussion by nepomuk16321** [eyeonus/Trade-Dangerous#170] [Q&A] [answered]
Is it possible to implement the “Age of data” switch for the “SELL” and “BUY” commands in the same way as for “RUN”?
![image](https://github.com/eyeonus/Trade-Dangerous/assets/20616775/3a09ebc7-21d4-451d-9aa3-59387fe0e726)

**Comment by eyeonus** [comment, marked as answer]
Interestingly, `buy` already has that. I've added it to `sell` now as well.
